app/controllers/AdminController.scala

```diff
@@ -102,29 +103,36 @@ class AdminController @Inject() (implicit val env: Environment[User, SessionAuth
     * @return
     */
   def getNeighborhoodCompletionRate = UserAwareAction.async { implicit request =>
-    if (isAdmin(request.identity)) {
-
-      // http://docs.geotools.org/latest/tutorials/geometry/geometrycrs.html
-      val CRSEpsg4326 = CRS.decode("epsg:4326")
-      val CRSEpsg26918 = CRS.decode("epsg:26918")
-      val transform = CRS.findMathTransform(CRSEpsg4326, CRSEpsg26918)
-
+    RegionCompletionTable.initializeRegionCompletionTable()
```

Comment:
Yes I had confirmed that this didn't run multiple times using the magic of print statements. You also know that it isn't because the call takes >10 seconds to run, but there are never any 10 second lags when calling the getNeighborhoodCompletionRate after the first call.